doc/guides/west/release-notes.rst

```diff
@@ -0,0 +1,116 @@
+West Release Notes
+##################
+
+v0.6.x
+******
+
+- No separate bootstrapper
+
+  In west v0.5.x, the program was split into two components, a bootstrapper and
+  a per-installation clone. See `Multiple Repository Management in the v1.14
+  documentation`_ for more details.
+
+  This is similar to how Google's Repo tool works, and lets west iterate quickly
+  at first. It caused confusion, however, and west is now stable enough to be
+  distributed entirely as one piece via PyPI.
+
+  From v0.6.x onwards, all of the core west commands and helper classes are
+  part of the west package distributed via PyPI. This eliminates complexity
+  and makes it possible to import west modules from anywhere in the system,
+  not just extension commands.
+- The ``selfupdate`` command still exists for backwards compatibility, but
+  now simply exits after printing an error message.
+- Manifest syntax changes
+
+  - A west manifest file's ``projects`` elements can now specify their fetch
+    URLs directly, like so:
+
+    .. code-block:: yaml
+
+       manifest:
+         projects:
+           - name: example-project-name
+             url: https://github.com/example/example-project
+
+    Project elements with ``url`` attributes set in this way may not also have
```

Comment:
Would it be an option to add an attribute `remote-name` that can be combined with the attribute `remote`, or the default remote, to form the final URL? The current proposal forces to use the attribute `url` in case two or more projects share the same repo.

As an example for PR #17629 the current west 0.6 implementation would result in:

```
    - name: lvgl_v5
      revision: pull/1/head
      path: modules/lib/gui/lvgl_v5
      url: https://github.com/zephyrproject-rtos/lvgl
    - name: lvgl_v6
      revision: pull/2/head
      path: modules/lib/gui/lvgl_v6
       url: https://github.com/zephyrproject-rtos/lvgl
```
If the `remote-name` attribute would be added this could become:
```
    - name: lvgl_v5
      revision: pull/1/head
      path: modules/lib/gui/lvgl_v5
      remote-name: lvgl
    - name: lvgl_v6
      revision: pull/2/head
      path: modules/lib/gui/lvgl_v6
      remote-name: lvgl
```

Reply:
> If the remote-name attribute would be added this could become:

How? Sorry, I don't understand the proposal. The `remote` attribute is already the name of the remote, so I'm not sure what `remote-name` means.

```
    - name: lvgl_v5
      revision: pull/1/head
      path: modules/lib/gui/lvgl_v5
      remote-name: lvgl
```

I'm not sure what this is doing; can you please explain? Do you perhaps mean `repo-name` instead of `remote-name`?

> The current proposal forces to use the attribute url in case two or more projects share the same repo.

Right, but what's the issue with that?

Reply:
Also, would you mind taking this to the west issue tracker? This PR is about documenting the release.

Reply:
> I'm not sure what this is doing; can you please explain? Do you perhaps mean `repo-name` instead of `remote-name`?

Yes, maybe that is a better naming for it, so `remote` + `repo-name` would form the URL.

> Right, but what's the issue with that?

Redundancy, is this not the reason why the remote concept was introduced?
In case more and more projects are introduced to support different API versions all of them need a hard coded `url`.

> Also, would you mind taking this to the west issue tracker? 

Ofcourse